argnorm/cli.py

```diff
@@ -30,5 +30,5 @@ def main():
         )
 
     prop_unmapped = ((result.ARO == 'ARO:nan').sum() + result.ARO.isna().sum()) / result.shape[0]
-    print(f'{round(1 - prop_unmapped, 3):.2%} args mapped.')
+    print(f'{args.output}:', f'{round(1 - prop_unmapped, 3):.2%} ARGs mapped.')
```

Comment:
This is not really related to removing unnecessary warnings, though

Reply:
Without the warnings it got a little difficult to see which file was getting normalized. I'd only see 'x% ARGs mapped'. Previously the warnings gave some idea as to which database was being used, but I wanted to make it clear for which file the normalization occurred.

Reply:
This makes sense. It should be mentioned in the commit message

Reply:
Done!